resources/build_resources.sh

```diff
@@ -28,11 +28,11 @@ function build_qt {
 }
 
 function build_ui {
-    build_qt "${PYTHON_BASE}/python ${PYTHON_BASE}/pyside-uic --from-imports" "$1.ui" "$1"
+    build_qt "pyside-uic --from-imports" "$1.ui" "$1"
```

Comment:
We can't rely on `pyside-rcc` and `pyside-uic` from the Shotgun Desktop in the future, as we won't be distributing PySide 1 anymore, so I'm relying on the one that is installed on my global Python.

Reply:
I imagine there is a lot more like this, but I guess we'll tackle them as and when we come to them.